app/controllers/reports_controller.rb

```diff
@@ -20,8 +22,33 @@ def create
     end
   end
 
+  def update_dropdown
+    org_ids = params[:org_ids]
+
+    @protocols = []
+    if org_ids.length > 1 
+      org_ids.each do |org_id|
+        @protocols << find_protocols(org_id)
+      end
+    else
+      @protocols << find_protocols(org_ids)
+    end
+    @protocols = @protocols.flatten
+
+  end
+
   private
 
+  def find_protocols(org_ids)
+    orgs = Array.wrap(Organization.find(org_ids))
```

Comment:
Can you explain the need for "Array.wrap"?  If you always want an array then maybe use "where" instead of "find".  "where" always returns an array even in the case of a single result.  "Organization.where(:id => org_ids)" would always return an array of organizations or an empty array